airbyte-integrations/connectors/destination-bigquery-denormalized/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDenormalizedGscDestinationAcceptanceTest.java

```diff
@@ -161,7 +161,7 @@ private Object getTypedFieldValue(final FieldValueList row, final Field field) {
     if (fieldValue.getValue() != null) {
       return switch (field.getType().getStandardType()) {
         case FLOAT64, NUMERIC -> fieldValue.getDoubleValue();
-        case INT64 -> fieldValue.getNumericValue().intValue();
+        case INT64 -> fieldValue.getLongValue();
```

Comment:
@aymen-k can you add a comment in top explaining why the int64 is converted to long? I think this will be useful.

Reply:
Yes sure, added in the last commit https://github.com/airbytehq/airbyte/pull/10241/commits/b3f778c6e8557f790ce2b9ceb25935682ae0b5a5